beacon-chain/p2p/subnets.go

```diff
@@ -208,10 +208,16 @@ func initializePersistentColumnSubnets(id enode.ID) error {
 	if ok && expTime.After(time.Now()) {
 		return nil
 	}
-	subs, err := computeSubscribedColumnSubnets(id)
+	subsMap, err := peerdas.CustodyColumnSubnets(id, params.BeaconConfig().CustodyRequirement)
 	if err != nil {
 		return err
 	}
+
+	subs := make([]uint64, 0, len(subsMap))
```

Comment:
Ideally, I think all subnets list should be converted at all levels into subnets maps.
As far as I know:
1. We want items in these lists not to have duplicates
2. We at some places look if an item is in the list (`O(n)`)

==> These 2 conditions makes a set to be a good candidate to store those data. Moreover lookup will go from `O(n)` to `O(1)`.